src/main/java/com/google/maps/internal/OkHttpPendingResult.java

```diff
@@ -194,6 +201,22 @@ private T parseResponse(OkHttpPendingResult<T, R> request, Response response) th
         return request.retry();
     }
 
+    byte[] bytes = getBytes(response);
+    R resp;
+    String contentType = response.header("Content-Type");
+
+    // Places Photo API special case
+    if (contentType != null &&
+        contentType.startsWith("image") &&
+        responseClass == PhotoRequest.Response.class &&
+        response.code() == 200) {
+      // Photo API response is just a raw image byte array.
+      PhotoResult result = new PhotoResult();
```

Comment:
As for raw `InputStream`, I'm unsure how to how to wedge that into `OkHttpPendingResult` such that all the retry and URL signing architecture continues to work. How about we look at that in follow up PR?